### PR TITLE
Updates to remove warnings in Xcode 12

### PR DESCRIPTION
**Pull request by dppeak**
This should eliminate the "Update to recommended settings" in Xcode 12.

### CODE REVIEW

File: QRoute.xcodeproj/xcshareddata/xcschemes/QRoute-watchOS.xcscheme [hunk 1]
```diff
@@ -38,8 +36,8 @@
             ReferencedContainer = "container:QRoute.xcodeproj">
          </BuildableReference>
       </MacroExpansion>
-      <AdditionalOptions>
-      </AdditionalOptions>
+      <Testables>
+      </Testables>
```
Comment (princeCoder) [inline QRoute.xcodeproj/xcshareddata/xcschemes/QRoute-watchOS.xcscheme:40]: Is this still useful?

Comment (dppeak) [reply QRoute.xcodeproj/xcshareddata/xcschemes/QRoute-watchOS.xcscheme:40]: Not understanding your question?

Comment (dppeak) [reply QRoute.xcodeproj/xcshareddata/xcschemes/QRoute-watchOS.xcscheme:40]: These are updates made by Xcode project.

Comment (quickthyme) [reply QRoute.xcodeproj/xcshareddata/xcschemes/QRoute-watchOS.xcscheme:40]: Xcode puts empty elements in the file, but it's not a concern.

**Review by quickthyme — APPROVED** [review]
LGTM

**Pull request merged**